Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/ChangeFeed/GetChangeFeedProcessorBuilderWithAllVersionsAndDeletesTests.cs

```diff
@@ -29,6 +32,128 @@ public async Task Cleanup()
             await base.TestCleanup();
         }
 
+        [TestMethod]
```

Comment:
```suggestion
        [TestMethod]
        [Timeout(30000)]
```

To avoid a bug causing an infinite wait, add a timeout to the test

Reply:
30 second (30000) timeout is not going to be enough as I explained before. The purger sometimes returns the purged document back in a matter of seconds, and other times, minutes. So, I will use it and maybe cap it a 5 minutes (300000 ms).

See
https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4523#discussion_r1625939998

and

https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4523#discussion_r1626535810

Reply:
fixed.